src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -171,15 +171,17 @@
                              :enumerate enumerate
                              :questions questions
                              :attributes working-attributes
-                             :editable? (and writer? curator?)
+                             :editable? (or writer? catandread?)
+                             :set-discoverable? (or can-share? catandread? owner?)
```

Comment:
This whole map is a prime candidate for my new `restructure` util.  `(utils/restructure foo bar baz)` gives `{:foo foo :bar bar :baz baz}`.